### DISCUSSION TITLE
Dayna SCSI with Ethernet instead of Wifi

### DISCUSSION THREAD

**Discussion by nehalem501** [BlueSCSI/BlueSCSI-v2#179] [Ideas]
Hello,
I was wondering if it would be possible to implement networking using Ethernet instead of Wi-Fi using a regular Raspberry Pi Pico and an Ethernet add-on card like described [here](https://www.raspberrypi.com/news/how-to-add-ethernet-to-raspberry-pi-pico/)?

**Comment by androda** [comment]
Unfortunately we don't have enough GPIO pins remaining to use that style of Ethernet chip.  Looks like RMII requires something like 7 pins and we only really have two available.  Those two pins are presently connected to the qwiic connector on Desktop BlueSCSI, so if you could find an Ethernet adapter which uses I2C then it would be potentially possible.
